lib/client/api.go

```diff
@@ -2525,13 +2482,8 @@ func (tc *TeleportClient) TransferFiles(ctx context.Context, hostLogin, nodeAddr
 	if !tc.Config.ProxySpecified() {
```

Comment:
If we've already attempted to connected to a cluster, isn't it a bit too late for this check?